include/Graph/Graph.hpp

```diff
@@ -2186,7 +2190,7 @@ namespace CXXGRAPH
 		}
 		return result;
 	}
-
+#ifdef WITH_COMPRESSION
```

Comment:
`readFromFile `and `writeToFile` functions, should have 2 different implementation for compression enabled, and disabled. We can't disable the `readFromFile `and `writeToFile` function only because no compression is set.